src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -26,25 +26,115 @@ public ScorerPipelineStep(Var<IDataView> data, Var<ITransformModel> model)
+    /// pipeline.Add(new TextLoader &lt;SentimentData&gt; (dataPath, separator: ","));
+    /// pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
+    /// pipeline.Add(new FastTreeBinaryClassifier());
+    /// 
+    /// var model = pipeline.Train&lt;SentimentData, SentimentPrediction&gt;();
+    /// </code>
+    /// </example>
+    /// </summary>
     [DebuggerTypeProxy(typeof(LearningPipelineDebugProxy))]
     public class LearningPipeline : ICollection<ILearningPipelineItem>
     {
         private List<ILearningPipelineItem> Items { get; } = new List<ILearningPipelineItem>();
 
+        /// <summary>
+        /// Construct an empty LearningPipeline object.
+        /// </summary>
         public LearningPipeline()
         {
         }
 
+        /// <summary>
+        /// Get the count of ML components in the LearningPipeline object
+        /// </summary>
         public int Count => Items.Count;
         public bool IsReadOnly => false;
+
+        /// <summary>
+        /// Add a data loader, transform or trainer into the pipeline. 
+        /// Possible data loader(s), transforms and trainers options are
+        /// <para>
+        /// Data Loader:
+        ///     <see cref="Microsoft.ML.TextLoader{TInput}" />
+        ///     etc.
+        /// </para>
+        /// <para>
+        /// Transforms:
+        ///     <see cref="Microsoft.ML.Transforms.Dictionarizer"/>,
+        ///     <see cref="Microsoft.ML.Transforms.CategoricalOneHotVectorizer"/>
+        ///     <see cref="Microsoft.ML.Transforms.MinMaxNormalizer"/>,
+        ///     <see cref="Microsoft.ML.Transforms.ColumnCopier"/>,
+        ///     <see cref="Microsoft.ML.Transforms.ColumnConcatenator"/>,
+        ///     <see cref="Microsoft.ML.Transforms.TextFeaturizer"/>,
+        ///     etc.
+        /// </para>
+        /// <para>
+        /// Trainers:
+        ///     <see cref="Microsoft.ML.Trainers.AveragedPerceptronBinaryClassifier"/>,
+        ///     <see cref="Microsoft.ML.Trainers.LogisticRegressor"/>,
+        ///     <see cref="Microsoft.ML.Trainers.StochasticDualCoordinateAscentClassifier"/>,
+        ///     <see cref="Microsoft.ML.Trainers.FastTreeRegressor"/>,
+        ///     etc.
+        /// </para>
+        /// For a complete list of transforms and trainers, please see "Microsoft.ML.Transforms" and "Microsoft.ML.Trainers" namespaces.
+        /// </summary>
+        /// <param name="item"></param>
         public void Add(ILearningPipelineItem item) => Items.Add(item);
+
+        /// <summary>
+        /// Remove all the transforms/trainers from the pipeline.
+        /// </summary>
         public void Clear() => Items.Clear();
+
+        /// <summary>
+        /// Check if a specific loader/transform/trainer is in the pipeline?
+        /// </summary>
+        /// <param name="item">Any ML component (data loader, transform or trainer) defined as ILearningPipelineItem.</param>
+        /// <returns>true/false</returns>
```

Comment:
>returns>true/false</returns [](start = 13, length = 27)

That the return value is either true or false is implied by the return value being `bool`. Can this be improved somehow? It seems like any "improvement" would just involve restating what what already written in the description, so is it acceptable in situations like this to just not include a `<returns>` tag?

Reply:
Again, I typically copy the MSDN docs:  https://msdn.microsoft.com/en-us/library/k5cf1d56(v=vs.110).aspx

> Return Value
Type: System.Boolean
true if item is found in the ICollection<T>; otherwise, false.